wca-guidelines.md

```diff
@@ -165,6 +165,7 @@ To be more informative, each Guideline is classified using one of the following
 - A3c3++) [CLARIFICATION] The competitor cannot "test" or "restart" the timer during inspection, since inspection ends once the competitor starts the solve (see [Regulation A4d](regulations:regulation:A4d)). If the competitor stops the timer after they have started it for the first time, this is considered stopping the solve (see [Regulation A6](regulations:regulation:A6))), even if they do so within 15 seconds of starting inspection.
 - A3c4+) [CLARIFICATION] If the use of a thin object inside the puzzle has been enforced, the judge must not remove the object on behalf of the competitor.
 - A3c4++) [CLARIFICATION] The competitor may remove the object during the solve, if they did not do it during the inspection (e.g. they forgot).
+- A4d+) [CLARIFICATION] If the competitor started the solve without waiting long enough for the green light to appear, the solve is disqualified (DNF).
```

Comment:
The competitor can only start the solve by starting the timer, so this doesn't really make sense. Could you rephrase this to describe what the competitor actually can't do (e.g. perform moves on the puzzles)?

Reply:
That's why I propose to change to 
> They start the solve by removing their hands from the timer.

Once you remove your hands - having started the timer or not - you can't go back and "re-start", because the judge is supposed to have stopped the stopwatch.

Reply:
What happens if the red light fails to turn on?

Alternative proposal: The competitor can retry starting the timer as long as the judge did not call 12 yet.